sdks/java/io/amazon-web-services2/src/main/java/org/apache/beam/sdk/io/aws2/options/AwsModule.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.beam.sdk.io.aws.options;
+package org.apache.beam.sdk.io.aws2.options;
```

Comment:
Don't we need to change a file path of this package (to `sdks/java/io/amazon-web-services2/src/main/java/org/apache/beam/sdk/io/aws2/...`) as well to reflect a package name?